source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,167 +1,209 @@
-#################################################################
+Mixin tool
+##########
 
-|CLOSIA| offers many curated bundles that you can install on your system to
-create your desired capabilities. If the available upstream bundles do not
-meet your needs, you can create and add your own custom bundles to your
-system using one of two methods. Note: Upstream refers to the official
-version of |CL|.
+The mixin tool, a light wrapper for :ref:`mixer`, is used to customize your
+installation of |CL-ATTR| by *side loading* a custom bundle. This is known
+as a *mixin*. Creating a mixin is useful when you need to add custom or 3rd
+party content but want to keep on the upstream update cycle. This guide covers
+the workflow of creating your own mixin using the mixin tool.
 
-The first method is to use the :ref:`mixer tool<mixer>` to create your own
-|CL| image and add your bundles to it.  Mixing your own |CL| image can
-give you great control and flexibility; however, you must act as an
-:abbr:`OSV (Operating System Vendor)` and maintain your releases and
-updates because you have forked from upstream.
+If you need to customize beyond what the mixin tool provides, use the
+:ref:`mixer tool<mixer>`.
 
-The second method is to use the :command:`mixin` tool, which also
-makes use of mixer to create custom bundles that you can add to your
-upstream |CL| system.  This  simpler method provides a “light” forking from
-upstream, which means you can continue to get upstream bundles and updates.
-If needed, you can easily revert your system back to the upstream version.
+Prerequisites
+*************
 
-This guide shows you how to accomplish the second method by following these
-steps:
+To start working with the mixin tool, you will need a recent image of |CL| with
+the `mixer` bundle installed. Before you install any new packages, update your
+OS with the following command:
 
-#. Set up the workspace.
-#. Copy your custom RPM package to the workspace.
-#. Create a bundle with your custom RPM package.
-#. Migrate your |CL| system to your custom mix.
-#. Add your custom bundle to your system.
-#. Optional: Revert your system back to 100% upstream.
+.. code-block:: bash
 
-Set up the workspace
-********************
+   sudo swupd update
 
-#. Install the mixer bundle to enable mixer.
+If the bundle is not yet installed, you can add it with the :command:`swupd bundle-add`
+command as follows:
 
-   .. code-block:: console
+.. code-block:: bash
 
-      $ sudo swupd bundle-add mixer
+   sudo swupd bundle-add mixer
 
-#. Create the workspace.
+Mixin workflow
+**************
 
-   .. code-block:: console
+The workflow to create your own mixin is outlined below.
 
-      $ sudo mkdir -p /usr/share/mix/local-rpms
+#. :ref:`create-workspace-mixin`
+#. :ref:`create-rpm`
+#. :ref:`copy-custom-rpm`
+#. :ref:`create-bundle`
+#. :ref:`migrate-mix`
+#. :ref:`add-bundle-to-system`
+#. :ref:`revert-to-upstream`
```

Comment:
The workflow should be this:
1. Add or create new repo(s).  
2. Add custom RPM packages to local repo (if applicable).
3. Add packages to bundles.
4. Build bundles.
5. Migrate your Clear Linux system to your custom mix.
6. Add your custom bundles to your system.
7. Optional: Revert your system back to upstream Clear.